src/HMI/tmplits.css

```diff
@@ -132,4 +132,55 @@ input::-webkit-outer-spin-button,
 input::-webkit-inner-spin-button {
   -webkit-appearance: none;
   margin: 0;
+}
+
+.tmplit-table {
+    width: 100%;
+    display: grid;
+    padding: 0px 20px 20px 20px;
+    overflow: auto;
+    grid-template-columns: var(--tmplit-table-grid-template-columns, 1fr 4fr 1fr);
+    text-align: var(--tmplit-table-text-align, left);
+    grid-template-rows: var(--tmplit-table-grid-template-rows, repeat(auto-fill, 50px));
+    height: var(--tmplit-table-height, 470px);
+    background-color: var(--tmplit-table-background-color, var(--md-sys-color-surface-container, #FFFFFF));
+}
+
+.tmplit-table-header {
+    font-size: var(--tmplit-table-header-font-size, 24px);
```

Comment:
For font values, it would be great to fallback to material design tokens first.
This makes it easier to keep the design similar across components.

Reply:
In this case I believe it would be something like `--md-sys-typescale-display-large-font`
Happy to dig into this with someone if we want to find the appropriate token.

Reply:
I like this idea. I'll need to generate a list of these to use. Do you know where to find these? I figure it would be a separate style sheet inside the css folder called "fonts"?

Reply:
thanks shane! material design them tool has diffent output in the figma plugin verse the web app